remoter-client/src/wrapper.r

```diff
@@ -47,16 +66,40 @@ run_step <- function(task_name, scdata, config) {
 send_output_to_api <- function(input, output) {
     c(config, plot_data = plotData) %<-% output
 
-    message("Sending to SNS topic", pipeline_config$sns_topic)
-    sns <- paws::sns()
-    result <- sns$publish(
-        Message = RJSONIO::toJSON(
-            list(
-                input = input,
-                config = config,
-                plot_data = plot_data
-            )
+
+    # upload output
+    s3 <- paws::s3(config=pipeline_config$aws_config)
+    id <- ids::uuid()
+    output <- RJSONIO::toJSON(
+        list(
+            config = config,
+            plotData = plot_data
+        )
+    )
+
+    message("Uploading results to S3 bucket", pipeline_config$results_bucket, " at key ", id, "...")
+    s3$put_object(
+        Bucket = pipeline_config$results_bucket,
+        Key = id,
+        Body = charToRaw(output)
+    )
+
+    message("Sending to SNS topic ", pipeline_config$sns_topic)
+    sns <- paws::sns(config=pipeline_config$aws_config)
```

Comment:
Slightly modified the results format to upload to S3 instead of copying the entire result into the SNS message. This is what the worker is currently doing for large responses, but I thought here it made sense to always do it as the responses we'd expect for plotting will be guaranteed to be bigger than the SNS message limit.